App/Sagas/NodeLifecycle.ts

```diff
@@ -23,6 +24,7 @@ export function * manageNode () {
 
       if (yield select(PreferencesSelectors.verboseUi)) {
         yield call(displayNotification, 'App State Change: ' + action.payload.newState)
+        yield call(logNewEvent, new Date().getTime(), 'App State Change', action.payload.newState)
```

Comment:
Should you move this outside the `if` here? Seems better that way so you don't miss some log data if you happen to turn off verbose UI.